tests/nougat/nut/git/status_spec.lua

```diff
@@ -60,6 +60,46 @@ describe("nut.git.status", function()
     t.eq(bar:generate(ctx), " ~2 -3 ")
   end)
 
+  it("works for vim-gitgutter", function()
+    package.loaded["gitsigns"] = {}
+
+    bar:add_item(nut.create({
+      prefix = " ",
+      content = {
+        nut.count("added", { prefix = "+", suffix = " " }),
+        nut.count("changed", { prefix = "~", suffix = " " }),
+        nut.count("removed", { prefix = "-", suffix = " " }),
+      },
+    }))
+
+    package.loaded["gitsigns"] = nil
+
+    -- mock data
+    vim.fn["exists"] = function(name)
+      return name == "*GitGutterGetHunkSummary"
```

Comment:
This will need to return a number
```suggestion
      return name == "*GitGutterGetHunkSummary" and 1 or 0
```